node/subsystem-util/src/lib.rs

```diff
@@ -601,10 +601,19 @@ impl<Spawner: SpawnNamed, Job: 'static + JobTrait> Jobs<Spawner, Job> {
 	}
 
 	/// Send a message to the appropriate job for this `parent_hash`.
+	/// Will only print a warning if the job is not running.
 	async fn send_msg(&mut self, parent_hash: Hash, msg: Job::ToJob) -> Result<(), Error> {
 		match self.running.get_mut(&parent_hash) {
 			Some(job) => job.send_msg(msg).await?,
-			None => return Err(Error::JobNotFound(parent_hash)),
+			None => {
+				// don't bring down the subsystem, this can happen to due a race condition
+				log::warn!(
```

Comment:
I am not even convinced that this should be a warning. I think Runtime API requests triggered by the Availability Distribution subsystem and others (in line with the assumptions outlined in the guide) would lead to serious warning spam.

Reply:
Even though Runtime API doesn't use this utility, my point is that sending messages with old relay-parents is fine and even expected in some cases. We should leave it up to the subsystem to handle those.